src/rdx/store.ts

```diff
@@ -1,6 +1,9 @@
-import { reducer } from "./reducer";
 import { configureStore } from "@reduxjs/toolkit";
+import thunkMiddleware from "redux-thunk";
+import loggerMiddleware from "redux-logger";
+import { reducer } from "./reducer";
 
 export const store = configureStore({
   reducer,
+  middleware: [thunkMiddleware, loggerMiddleware],
```

Comment:
loggerMiddleware only for dev?

Reply:
Any ideas how we should do it?

Reply:
https://github.com/LogRocket/redux-logger#log-only-in-development

Reply:
done